lit-tailwind.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Using tailwindcss with web components and ShadowDOM in LitElement
+domain: campzulu.hashnode.dev
+tags: web-development, tailwind-css, litelement, web-components, typescript, vite
+cover: https://cdn.hashnode.com/res/hashnode/image/unsplash/Im7lZjxeLhg/upload/v1666377414880/YI2XLcTuz.jpeg?w=1600&h=840&fit=crop&crop=entropy&auto=compress,format&format=webp
+publishAs: r3dDoX
+ignorePost: true
+---
+
+We recently wanted to use tailwindcss to help developers at one of our customers create maintainable
+and scalable styling. At the same time we were creating web components for a design system. This
+raises one big issue with tailwind since we were going to use ShadowDOM. This will prevent globally
+available tailwind utility classes to be available in our components. But with LitElement we found a
```

Comment:
Maybe a small exert why this isn't possible would be nice.

Reply:
Added a small chapter "The problem with ShadowDOM" to explain the issue in more depth 👍

Reply:
Nice one!